artifactory/services/utils/aqlquerybuilder.go

```diff
@@ -197,22 +197,29 @@ func buildInnerArchiveQueryPart(triple RepoPathFile, archivePath, archiveName st
 	return fmt.Sprintf(innerQueryPattern, getAqlValue(triple.repo), getAqlValue(triple.path), getAqlValue(triple.file), getAqlValue(archivePath), getAqlValue(archiveName))
 }
 
-func buildExcludeQueryPart(excludePatterns []string, useLocalPath, recursive bool) string {
-	if excludePatterns == nil {
-		return ""
-	}
+func buildExcludeQueryPart(params *ArtifactoryCommonParams, useLocalPath, recursive bool) string {
 	excludeQuery := ""
-	var excludePairs []RepoPathFile
-	for _, excludePattern := range excludePatterns {
-		excludePairs = append(excludePairs, createPathFilePairs("", prepareSearchPattern(excludePattern, false), recursive)...)
+	var excludeTriples []RepoPathFile
+	if len(params.GetExclusions()) > 0 {
+		for _, excludePattern := range params.GetExclusions() {
+			excludeTriples = append(excludeTriples, createRepoPathFileTriples(prepareSearchPattern(excludePattern, true), recursive)...)
+		}
+	} else {
+		for _, excludePattern := range params.GetExcludePatterns() {
```

Comment:
Let's add a comment above this line, saying that 'excludePatterns' are deprecated and replaced replaced by 'exclusions'